swift/Rakefile

```diff
@@ -97,10 +97,11 @@ end
 def create_project(name, bindist)
     project = Xcodeproj::Project.new(name)
 
-    project.build_configurations.each do |conf|
-        conf.build_settings["CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER"] = "YES"
+    project.build_configurations.each do |config|
+        config.build_settings["CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER"] = "YES"
         # Disable header maps as they cause problems with parallel builds on Xcode 12
-        conf.build_settings["USE_HEADERMAP"] = "NO"
+        config.build_settings["USE_HEADERMAP"] = "NO"
+        config.build_settings["DEVELOPMENT_TEAM"] = "U4TBVKNQ7F";
```

Comment:
Why do need to bake in the dev team now?

Reply:
Iet me try again without it, it might not be required after all.